analyzers/tests/SonarAnalyzer.Test/TestCases/SymbolicExecution/Roslyn/ConditionEvaluatesToConstant.CSharp12.cs

```diff
@@ -8,28 +8,45 @@ void CollectionExpressions(int[] array)
         int[] knownLength = [1, 2, 3, 4, 5, 6, 7, 8, 9, 10];
         for (var i = 0; i < knownLength.Length; i++)
         {
-            if (i > 5)                  // Compliant
+            if (i > 5)                      // Compliant
             {
                 Console.WriteLine(i);
             }
         }
+        if (knownLength.Length == 0) { }    // FN
```

Comment:
This should not be part of this humongous PR 😄

Reply:
What do you mean exactly? I fixed the tests, which allowed me to use `.Verify`.